<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcoupledialog</class>
 <widget class="QDialog" name="addcoupledialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nouveau couple</string>
  </property>
  <layout class="QVBoxLayout" name="NouveauCouple">
   <item>
    <layout class="QHBoxLayout" name="relationLayout">
     <item>
      <widget class="QComboBox" name="comboRelation">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="isOrientedLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="note1Layout">
     <item>
      <widget class="QLabel" name="labelNote1">
       <property name="text">
        <string>Note 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboNote1"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="note2Layout">
     <item>
      <widget class="QLabel" name="labelNote2">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Note 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboNote2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="labelLayout">
     <item>
      <widget class="QLabel" name="labelCouple">
       <property name="text">
        <string>Label du couple</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="labelEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="notificationLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="submitButton">
     <property name="text">
      <string>Ajouter le couple</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
